e2e-identity/tests/e2e.rs

```diff
@@ -111,16 +106,16 @@ mod alg {
 
     #[tokio::test]
     async fn ed25519_should_succeed() {
-        let test = E2eTest::new_internal(false, JwsAlgorithm::Ed25519, OidcProvider::Dex)
-            .start(docker())
+        let test = E2eTest::new_internal(false, JwsAlgorithm::Ed25519, OidcProvider::Keycloak)
+            .start()
```

Comment:
What's the reason we're switching to Keycloak here?

Reply:
Thanks for taking the time to pull this into a separate commit and writing a nice commit message!